### DISCUSSION TITLE
duplicate existing github repo to new github site?

### DISCUSSION THREAD

**Discussion by jsarey** [community/community#33607] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
I would like to understand the recommended way to duplicate/link/mirror an existing github repository so that an up-to-date version appears on a github site hosted by another organization or user. In other words, I would like to have:
github.com/organizationA/project1
mirrored or linked so that it also appears at:
github.com/organizationB/project1
I would have "owner" or other appropriate access privileges on both sites.
Can I do this? Would this be (best) achieved by following the instructions on duplicating a repository?
https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository
Thanks

**Comment by airtower-luna** [comment, marked as answer]
Those instructions should work, yes. They're tailored for mirroring all branches, if you don't need (or want) that you can simply add remotes for both GitHub repositories on any local copy and push the branch(es) you want.

If you want to *keep* the repositories in sync without manual intervention it gets a bit more complicated. The [are some custom actions for that](https://github.com/repo-sync), but I haven't tried them myself.

**Comment by bharathikannan25** [comment]
Need new repository for IBM-EPBL,cause the old invitation got revoked, so help me to get new one. Please
